ubuntu/16.04/usr/local/bin/supervisor_start

```diff
@@ -8,6 +8,10 @@ source /usr/local/share/bootstrap/setup.sh
 source /usr/local/share/bootstrap/run_confd.sh
 
 # Initialisation - Runtime installation tasks
+for file in /usr/local/bin/supervisor_custom_start-*
```

Comment:
yes, it'll use the filesystem's sort order, the reason for the 20's

10 - web server
20 - framework